### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by SvetlanaMikheeva**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -3,29 +3,59 @@
...
-# How to use ASPxUploadControl to upload an image and display it on a web page
+This example demonstrates how to use the [Upload Control](https://docs.devexpress.com/AspNet/8298/components/file-management/file-upload) to upload an image to the server and display the uploaded image on a web page.
+
+![Upload and Display an Image](upload-image.png)
+
+## Overview
+
+Follow the steps below to display an image on a web page once a user uploads the image to the server:
+
+1. Create the *Images* folder in the root folder of your project. In the newly created folder, place an image that you want to initially display.
```
Comment (dmitry-eliseev-devexpress) [inline Readme.md]: Word order -- "initially" goes to the end of the base sentence as supplement (an aspect of circumstances that answers the question "how"):

```suggestion
1. Create the *Images* folder in the root folder of your project. In the newly created folder, place an image you want to display initially.
```

File: Readme.md [hunk 2]
```diff
@@ -3,29 +3,59 @@
...
 
+4. Handle the Upload Control's server-side [FileUploadComplete](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxUploadControl.FileUploadComplete?p=netframework) event. The [UploadedFile](https://docs.devexpress.com/AspNet/DevExpress.Web.FileUploadCompleteEventArgs.UploadedFile?p=netframework) event argument contains the uploaded image. Call the image's [SaveAs](https://docs.devexpress.com/AspNet/DevExpress.Web.UploadedFile.SaveAs(System.String)) method to save the image on the server. Set the [CallbackData](https://docs.devexpress.com/AspNet/DevExpress.Web.FileUploadCompleteEventArgs.CallbackData) property to the relative image path to pass it to the client.
 
+    ```csharp
+    protected void Upload_FileUploadComplete(object sender, FileUploadCompleteEventArgs e) {
+        // Assign a relative path to the 'CallbackData' property
+        // Relative path: "Images\Picture2023-02-17 10-19-58.jpg"
+        e.CallbackData = String.Format("Images\\Picture{0}.jpg", DateTime.Now.ToString("yyyy-MM-dd hh-mm-ss"));
+        
+        // Send a full path as the 'path' parameter to the 'SaveAs' method
```
Comment (dmitry-eliseev-devexpress) [inline Readme.md]: We would send something to the connected server, however, we can pass a full path to the `SaveAs` method...

Word order.

Again, I moved the aspect of circumstances to the end of the base sentence structure. Pass a full path to the method how? As the `path" parameter.

```suggestion
        // Pass a full path to the 'SaveAs' method as the 'path' parameter
```

File: VB/WebSite/Default.aspx.vb [hunk 3]
```diff
@@ -8,8 +8,13 @@ Imports System.Web.UI
 Partial Public Class _Default
 	Inherits Page
 	Protected Sub Upload_FileUploadComplete(ByVal sender As Object, ByVal e As FileUploadCompleteEventArgs)
+		' Assign a relative path to the 'CallbackData' property
+        ' Relative path: "Images\Picture2023-02-17 10-19-58.jpg"
 		e.CallbackData = String.Format("Images\Picture{0}.jpg", DateTime.Now.ToString("yyyy-MM-dd hh-mm-ss"))
+		
+		' Send a full path as the 'path' parameter to the 'SaveAs' method
```
Comment (dmitry-eliseev-devexpress) [inline VB/WebSite/Default.aspx.vb]: Likewise.

```suggestion
		' Pass a full path to the 'SaveAs' method as the 'path' parameter
```

**Review by dmitry-eliseev-devexpress — APPROVED** [review]
This is a good and clear topic. However, please pay more attention to the word order. Generally, it's not the best idea to break the base sentence structure with supplementary information.

File: Readme.md [hunk 4]
```diff
@@ -3,29 +3,59 @@
...
+
+![Upload and Display an Image](upload-image.png)
+
+## Overview
+
+Follow the steps below to display an image on a web page once a user uploads the image to the server:
+
+1. Create the *Images* folder in the root folder of your project. In the newly created folder, place an image you want to display initially.
+
+2. Add the [HTML \<img\> tag](https://www.w3schools.com/tags/tag_img.asp) to a page. Use the tag's **id** attribute to specify the tag identifier. Assign the path to the initially displayed image to the tag's **src** attribute. The **alt** attribute allows you to specify an alternate text for the image.
```
Comment (arminal) [inline Readme.md]:
```suggestion
2. Add the [HTML \<img\> tag](https://www.w3schools.com/tags/tag_img.asp) to a page. Use the tag's **id** attribute to specify the tag identifier. Assign the path that runs to the initially displayed image to the tag's **src** attribute. The **alt** attribute allows you to specify an alternate text for the image.
```
Can you carefully review this edit? I want to make sure that the technical meaning is not altered.

Comment (SvetlanaMikheeva) [reply Readme.md]: The meaning is not altered, thank you.

**Pull request merged**